src/RaisedButton/RaisedButton.js

```diff
@@ -134,6 +134,13 @@ class RaisedButton extends Component {
      */
     className: PropTypes.string,
     /**
+     * An element like Link tag or a string can be used inside RaisedButton.
```

Comment:
I think that we should make the description generic, i.e. always the same in proptypes documentatino generator.
We have that one on the `GridTitle` that's a bit too verbose but more explicit:
```jsx
     /**
     * Either a string used as tag name for the tile root element, or a ReactElement.
     * This is useful when you have, for example, a custom implementation of
     * a navigation link (that knows about your routes) and you want to use it as the primary tile action.
     * In case you pass a ReactElement, please ensure that it passes all props,
     * accepts styles overrides and render it's children.
     */
```